config/locales/cookies.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  cookies:
+    manage:
+      title: Manage teacher training applications
+    apply:
+      title: Apply for teacher training
```

Comment:
Do we need this localisation file, given there’s the `service_name` helper, and we already have these strings localised?

Reply:
`service_name` is used by the cookie banner but not the cookies page. 

Settings this up enables me to set `@application` to `:apply` and have the correct name retrieved from the locale, as well as the consented-to cookie named appropriately. 

I could probably reuse the locale on the cookies banner though rather than passing in the service name.